site/content/presentations-material.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Presentations Material"
+menu = "main"
+weight = 620
++++
+
+In this page you can find some links to presentations material about Eclipse Hono showed during events, conferences and meetups.
```

Comment:
what about
`On this page you can find links to arbitrary material covering Eclipse Hono which has been presented at events, conferences and meetups.`